cdap-ui/app/directives/navbar/namespace.html

```diff
@@ -7,7 +7,7 @@
       </a>
     </li>
     <li role="menuitem" ng-show="namespaces.length === 1" class="disabled">
-      <a href=""> <strong> No more Namespaces </strong> </a>
+      <a href="">No other namespaces</a>
```

Comment:
Hmm, @ajainarayanan, has the design for this been finalized? Would it be better instead that we disable the drop down when there are no namespaces?